cmd/controller/app/options/options.go

```diff
@@ -143,6 +146,8 @@ const (
 	defaultPrometheusMetricsServerAddress = "0.0.0.0:9402"
 
 	defaultDNS01CheckRetryPeriod = 10 * time.Second
+
+	defaultIssuerSetupTimeout = 10 * time.Second
```

Comment:
I think it would. I don't know how the original value was decided upon but I'd guess it was just someone saying "10 seconds will probably be fine".

I don't know of a workflow where people are creating `Issuer` resources so frequently that we need to be conservative with this timeout. The default could go up by a _lot_ as far as I'm concerned.

Reply:
Probably doesn't get created that often, but in the case where one has a slow responding ACME API, you'll be scratching a huge hole in your head trying to figure out why it doesn't work out of the box. So a higher default would be nice, then you don't need to add notes saying you need to configure the timeout